providers/alternatives.rb

```diff
@@ -38,8 +38,9 @@
         description = "Add alternative for #{cmd}"
         converge_by(description) do
           Chef::Log.debug "Adding alternative for #{cmd}"
-          shell_out("rm /var/lib/alternatives/#{cmd}")
```

Comment:
you are removing what is there, but debian manages this all for you. http://linux.die.net/man/8/update-alternatives